packages/orbit-components/src/HorizontalScroll/index.tsx

```diff
@@ -11,8 +11,6 @@ import ChevronBackward from "../icons/ChevronBackward";
 import ChevronForward from "../icons/ChevronForward";
 import type { Props, ScrollSnap } from "./types";
 
-const TRIGGER_OFFSET = 20;
```

Comment:
I'm still curious why we added this in the first place and can't see any new glitch by removing it… So I guess it will have to work for now